app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -97,19 +101,34 @@
     </div>
 
     <div class="form-group" >
-      <label class="required">Access Mode</label><br/>
-      <label class="radio-inline">
-        <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadWriteOnce" aria-describedby="access-modes-help" ng-checked="true" >
-        Single User (RWO)
-      </label>
-      <label class="radio-inline">
-        <input type="radio" id="accessModes" name="accessModes" ng-model="claim.accessModes" value="ReadWriteMany" aria-describedby="access-modes-help" >
-        Shared Access (RWX)
-      </label>
-      <label class="radio-inline">
-        <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadOnlyMany" aria-describedby="access-modes-help">
-        Read Only (ROX)
+      <label class="required">Access Mode
+        <span ng-if="claim.storageClass && (claim.storageClass | storageClassAccessMode)">
+          <small>(cannot be changed)</small>
+        </span>
       </label>
+      <div>
+        <label class="radio-inline">
+          <input type="radio" name="accessModes" ng-model="claim.accessModes" value="ReadWriteOnce"
+                 aria-describedby="access-modes-help" ng-true-value="’1’" ng-false-value="’0’"
+                 ng-checked="((claim.storageClass | storageClassAccessMode)==='ReadWriteOnce') || !claim.storageClass || (claim.storageClass.metadata.name === 'No Storage Class')"
```

Comment:
Will this prevent me from selecting another value?

Reply:
@spadgett - You can select other values if the access mode is not defined.  You are right about a default not being selected for a storage class without a access mode.  I have updated my test to include a storage class like that and I have fixed the problem.